tests/Doctrine/Tests/DBAL/Functional/Schema/SQLAnywhereSchemaManagerTest.php

```diff
@@ -24,7 +24,7 @@ public function testCreateAndListViews()
         self::assertCount(1, $views, 'Database has to have one view.');
         self::assertInstanceOf(View::class, $views[$name]);
         self::assertEquals($name, $views[$name]->getName());
-        self::assertEquals($sql, $views[$name]->getSql());
+        self::assertRegExp('/^SELECT \* from "?DBA"?\."?view_test_table"?$/', $views[$name]->getSql());
```

Comment:
@morozov I am not happy with this assertion either but I am not sure what the better alternative would be. Basically we just want to know whether the schema manager can introspect view definitions. Thoughts?

Reply:
> Basically we just want to know whether the schema manager can introspect view definitions.

How is this definition going to be used down the line? E.g. if the view is part of the schema managed by an application, and during a migration, we want to see if the view definition hasn't changed, this inconsistency may be a problem, since the string-to-string comparison may or may not match depending on the database and the way how the view is defined on the application side.